tests/integ/test_workflow.py

```diff
@@ -67,7 +67,6 @@
     ConditionLessThanOrEqualTo,
 )
 from sagemaker.workflow.condition_step import ConditionStep
-from sagemaker.workflow.condition_step import JsonGet as ConditionStepJsonGet
```

Comment:
The JsonGet defined in `condition_step.py` is deprecated so update to the latest one